netutils/config/parser.py

```diff
@@ -328,7 +345,12 @@ def __init__(self, config):
         super(BaseBraceConfigParser, self).__init__(config)
 
     @property
-    def config_lines_only(self):
+    def banner_end(self) -> str:
```

Comment:
Same as previous

Reply:
Sorry I see now what you mean, I removed these.